Tests/Responsible.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
```

Comment:
fyi this should not be a requirement to be able to use stryker v1 :)

Reply:
Thanks for pointing it out! However, ReproducibleBuilds requires .NET 5.0.300 SDK or newer, so targeting .NET 5 in my **tests** simplifies the GH Actions runner set up.

Reply:
Mhm I had not realized Reproducible Builds required such a new SDK. Weird.